src/v2/Apps/Artwork/Server/handleArtworkImageDownload.ts

```diff
@@ -14,9 +14,5 @@ export const handleArtworkImageDownload = async (req, res, next) => {
       imageRequest.set("X-ACCESS-TOKEN", req.user.get("accessToken"))
     }
     req.pipe(imageRequest, { end: false }).pipe(res)
-  } else {
```

Comment:
which error are we sending back to the user if he's not authorized to download an artwork image?

Reply:
This is no longer relevant because we only show the download image button to authorized (admin) users. It's hidden for everyone else. (This code is superrrr old.)